### PR TITLE
Use assertj fluent style in flowable-rest module.

### PR DESCRIPTION
**Pull request by dbmalkovsky**
The module had a mix of assertion styles.  This is part 6 for this module.
This finishes the `org.flowable.rest.service.api.runtime` package.

### CODE REVIEW

File: modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java [hunk 1]
```diff
@@ -118,12 +107,13 @@ public void testCreateIdentityLink() throws Exception {
             CloseableHttpResponse response = executeRequest(httpPost, HttpStatus.SC_CREATED);
             JsonNode responseNode = objectMapper.readTree(response.getEntity().getContent());
             closeResponse(response);
-            assertNotNull(responseNode);
-            assertEquals("kermit", responseNode.get("user").textValue());
-            assertEquals("myType", responseNode.get("type").textValue());
-            assertTrue(responseNode.get("group").isNull());
-            assertTrue(responseNode.get("url").textValue()
-                    .endsWith(RestUrls.createRelativeResourceUrl(RestUrls.URL_TASK_IDENTITYLINK, task.getId(), RestUrls.SEGMENT_IDENTITYLINKS_FAMILY_USERS, "kermit", "myType")));
+            assertThat(responseNode).isNotNull();
```
Comment (filiphr) [inline modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java:110]: Why not `assertThatJson` here as well?

File: modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java [hunk 2]
```diff
@@ -134,12 +124,13 @@ public void testCreateIdentityLink() throws Exception {
             response = executeRequest(httpPost, HttpStatus.SC_CREATED);
             responseNode = objectMapper.readTree(response.getEntity().getContent());
             closeResponse(response);
-            assertNotNull(responseNode);
-            assertEquals("sales", responseNode.get("group").textValue());
-            assertEquals("myType", responseNode.get("type").textValue());
-            assertTrue(responseNode.get("user").isNull());
-            assertTrue(responseNode.get("url").textValue()
-                    .endsWith(RestUrls.createRelativeResourceUrl(RestUrls.URL_TASK_IDENTITYLINK, task.getId(), RestUrls.SEGMENT_IDENTITYLINKS_FAMILY_GROUPS, "sales", "myType")));
+            assertThat(responseNode).isNotNull();
```
Comment (filiphr) [inline modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java:127]: Why not `assertThatJson` here as well?

File: modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java [hunk 3]
```diff
@@ -188,19 +179,23 @@ public void testGetSingleIdentityLink() throws Exception {
...
             CloseableHttpResponse response = executeRequest(httpGet, HttpStatus.SC_OK);
             JsonNode responseNode = objectMapper.readTree(response.getEntity().getContent());
             closeResponse(response);
-            assertNotNull(responseNode);
-            assertEquals("kermit", responseNode.get("user").textValue());
-            assertEquals("myType", responseNode.get("type").textValue());
-            assertTrue(responseNode.get("group").isNull());
-            assertTrue(responseNode.get("url").textValue()
-                    .endsWith(RestUrls.createRelativeResourceUrl(RestUrls.URL_TASK_IDENTITYLINK, task.getId(), RestUrls.SEGMENT_IDENTITYLINKS_FAMILY_USERS, "kermit", "myType")));
+            assertThat(responseNode).isNotNull();
```
Comment (filiphr) [inline modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java:187]: Why not `assertThatJson` here as well?

File: modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java (hunk 1 above)
Comment (dbmalkovsky) [reply modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java:110]: my bad; fixed.

File: modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java (hunk 2 above)
Comment (dbmalkovsky) [reply modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java:130]: my bad; fixed.

File: modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java (hunk 3 above)
Comment (dbmalkovsky) [reply modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/TaskIdentityLinkResourceTest.java:193]: my bad; fixed.  Guess I was too tired.

**Pull request merged**